milner_desvars/gazebo.md

```diff
@@ -0,0 +1,71 @@
+#Notes
+
+##Client and server separation
+
+When you run Gazebo you are actually running to different executables. In one hand "gzserver" and in the other "gzclient".
```

Comment:
You can use `this Markdown style` instead for `gzserver` and `gzclient`.

In order to do that, use this:

```
`this is a one-line code`
```